aiida/cmdline/commands/cmd_node.py

```diff
@@ -61,6 +64,87 @@ def repo_ls(node, relative_path, color):
         echo.echo_critical(exception)
 
 
+@verdi_node_repo.command('dump')
+@click.option(
+    '-f',
+    '--force',
+    help='Overwrite existing directories and files without prompting for confirmation.',
+    flag_value=True
+)
+@arguments.NODE()
+@click.argument(
+    'output_directory',
+    type=click.Path(exists=True, file_okay=False, writable=True),
+    required=True,
+)
+@with_dbenv()
+def repo_dump(node, output_directory, force):
+    """Copy the repository files of a node to an output directory."""
+    from aiida.orm.utils.repository import FileType
+
+    output_directory = pathlib.Path(output_directory)
+
+    output_directory.mkdir(exist_ok=True)
+
+    def _copy_tree(key, output_dir):  # pylint: disable=too-many-branches
+        for file in node.list_objects(key=key):
+            if not key:
+                file_path = file.name
+            else:
+                file_path = key + '/' + file.name
+            if file.type == FileType.DIRECTORY:
+                new_out_dir = output_dir / file.name
+                if new_out_dir.exists():
+                    if force:
+                        prompt_res = 'o'
+                    else:
+                        prompt_res = click.prompt(
+                            "Directory '{}' exists. Do you wish to abort [a], "
+                            'skip this directory [s], merge directory contents [m], '
+                            'or overwrite the directory (delete current contents) [o]?'.format(new_out_dir),
```

Comment:
```suggestion
                            'or replace [r]?'.format(new_out_dir),
```
If we use "replace [r]" instead of "overwrite" we don't need the clarification about the contents.